<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowTitle">
   <string>Изменение цены</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>

  <layout class="QVBoxLayout">

   <item>
    <layout class="QFormLayout">
     <item row="0" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Старая цена</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="oldPriceEdit"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Новая цена</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="newPriceEdit"/>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QPushButton" name="calcButton">
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
</ui>
